main/http_server/axe-os/src/app/pipes/uptime-since.pipe.ts

```diff
@@ -16,9 +16,9 @@ export class UptimeSincePipe implements PipeTransform {
       const s = Math.floor(seconds % 60)
       
       const dDisplay = d > 0 ? d + ( "d" ) : ""
-      const hDisplay = h > 0 ? h + ( "h" ) : ""
-      const mDisplay = m > 0 ? m + ( "m" ) : ""
-      const sDisplay = s > 0 ? s + ( "s" ) : ""
+      const hDisplay = h > 0 || d > 0 ? h + ( "h" ) : ""
+      const mDisplay = m > 0 || h > 0 ? m + ( "m" ) : ""
+      const sDisplay = s > 0 || m > 0 ? s + ( "s" ) : ""
```

Comment:
Hmm, not 100% what you are going for here, but you might want something like this:

```
     const dDisplay = d > 0 ? d + ( "d" ) : ""
      const hDisplay = h > 0 || d > 0 ? h + ( "h" ) : ""
      const mDisplay = m > 0 || h > 0 || d > 0 ? m + ( "m" ) : ""
      const sDisplay = s > 0 || m > 0 || h > 0 || d > 0 ? s + ( "s" ) : ""
      ```
      
Unless you do want to hide minutes when it and hours are 0

Reply:
Actually, this may be a little cleaner:

```
      const dDisplay = d > 0 ? d + ( "d" ) : ""
      const hDisplay = h > 0 || dDisplay.length > 0 ? h + ( "h" ) : ""
      const mDisplay = m > 0 || hDisplay.length > 0 ? m + ( "m" ) : ""
      const sDisplay = s > 0 || mDisplay.length > 0 ? s + ( "s" ) : ""
```

Reply:
> const dDisplay = d > 0 ? d + ( "d" ) : ""
>       const hDisplay = h > 0 || dDisplay.length > 0 ? h + ( "h" ) : ""
>       const mDisplay = m > 0 || hDisplay.length > 0 ? m + ( "m" ) : ""
>       const sDisplay = s > 0 || mDisplay.length > 0 ? s + ( "s" ) : ""

Good suggestion, using .length would work better. I will update the PR to use this logic. Thank you!